pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -268,6 +273,44 @@ func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.Op
 	return false, nil
 }
 
+func (c *InstallerController) updateRevisionStatus(operatorStatus *operatorv1.StaticPodOperatorStatus) error {
+	failedRevisions := make(map[int32]struct{})
+	currentRevisions := make(map[int32]struct{})
+	for _, nodeState := range operatorStatus.NodeStatuses {
+		failedRevisions[nodeState.LastFailedRevision] = struct{}{}
+		currentRevisions[nodeState.CurrentRevision] = struct{}{}
+	}
+	delete(failedRevisions, 0)
+
+	// If all current revisions point to the same revision, then mark it successful
+	if len(currentRevisions) == 1 {
+		err := c.updateConfigMapForRevision(currentRevisions, string(corev1.PodSucceeded))
+		if err != nil {
+			return err
+		}
+	}
+	return c.updateConfigMapForRevision(failedRevisions, string(corev1.PodFailed))
+}
+
+func (c *InstallerController) updateConfigMapForRevision(currentRevisions map[int32]struct{}, phase string) error {
+	for currentRevision := range currentRevisions {
+		statusConfigMap, err := c.kubeClient.CoreV1().ConfigMaps(c.targetNamespace).Get(statusConfigMapNameForRevision(currentRevision), metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		if err != nil {
+			return err
+		}
+		statusConfigMap.Data["revision"] = fmt.Sprintf("%d", currentRevision)
```

Comment:
This looks unnecessary.  Particular since we do not expect to mount the configmap.